sim/src/libuart.c

```diff
@@ -37,43 +31,33 @@ ivv-itc@lists.nasa.gov
 /* usart device handles */
 static NE_Uart *usart_device[NUM_USARTS] = {0};
 
-/* usart mutex */
-static uint32 nos_usart_mutex = 0;
-
 /* public prototypes */
 void nos_init_usart_link(void);
 void nos_destroy_usart_link(void);
+int32_t sim_MutSemCreate (pthread_mutex_t *mutex, uint32_t options);
+int32_t sim_MutSemDelete (pthread_mutex_t *mutex);
```

Comment:
What are these doing? I tested the code and it all appears to work as intended, but when I searched through the nos3 repo, I did not see anywhere that these sim_MutSem* are being used (especially as the mutexes have been removed from a lot of the other things, which makes sense).

Reply:
Agree... these should have been deleted.